### PR TITLE
Idir gold only no teams

### PR DESCRIPTION
**Pull request by JafferFreshWorks**

### CODE REVIEW

File: cypress/integration/3.IDIR-only-GOLD-no-Teams.feature [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        #Below is reusable line for validation by passing values
+        Then User validates information "Project Name" value is "CSS-HappyPath-end2end_Gold"
+        And User validates information "Identity Providers Required" value is "idir"
+        And User validates information "Dev Redirect URIs" value is "https://localhost:3000"
+        And User clicks on "Submit" button
+        And User clicks on "Confirm" button
+        And User Verfiy "Status" is "Submitted" in Table
+        And User Verfiy "Service Type" is "Gold" in Table
+        And User waits for "20" minutes
+        And User clicks on "Download" button at position "1"
```
Comment (junminahn) [inline cypress/integration/3.IDIR-only-GOLD-no-Teams.feature:53]: I believe the most benefits of BDD is the readibility and `position "1"` does not explain anything to me.

Comment (junminahn) [inline cypress/integration/3.IDIR-only-GOLD-no-Teams.feature:53]: I believe the most benefits of BDD is the readibility and `position "1"` does not explain anything to me.

File: cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
+var a = 0;
```
Comment (junminahn) [inline cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js]: `a` and `b`? variable names can be much more meaningful.

Comment (junminahn) [inline cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js]: `a` and `b`? variable names can be much more meaningful.

File: cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+var b = 1;
+
+Then("User clicks on {string} tab on screen", async (TabName) => {
+    cy.wait(1000);
+    a = 0;
+    b = 1;
+    await cy.xpath("//*[@id='__next']/div[1]/div[2]//a[text()='" + TabName + "']").click();
```
Comment (junminahn) [inline cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js:8]: do you need a `test id` for the table?

Comment (junminahn) [inline cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js:8]: do you need a `test id` for the table?

File: cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+And('User clicks on {string} button at position {string}', async function (buttonName, position) {
+    // await cy.xpath(Request_SSO_Integration.PlusrequestSSOIntegrationBtn()).click();
+    await cy.xpath("(//button[contains(.,'" + buttonName + "')])[" + position +  "]").click();
+    await cy.wait(2000);
+  });
+
+And("User Enters {string} on {string} textbox at row {string} on {string} pop-up", async (Value, TextboxName, RowNo, PopUpName) => {
+    cy.wait(500);
+    if (TextboxName == 'Role Name') {
```
Comment (junminahn) [inline cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js]: in javascript, strict comparison is preferred; `===` intead of `==`.

Comment (junminahn) [inline cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js]: in javascript, strict comparison is preferred; `===` intead of `==`.

File: cypress/integration/3.IDIR-only-GOLD-no-Teams.feature (hunk 1 above)
Comment (JafferFreshWorks) [reply cypress/integration/3.IDIR-only-GOLD-no-Teams.feature:53]: There are 3 downloads on the screen and I chose the Position 1 download for our current scenario.

File: cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js (hunk 2 above)
Comment (JafferFreshWorks) [reply cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js]: I've used it as a Flag and further used as a counter.

Comment (JafferFreshWorks) [reply cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js]: I've used it as a Flag and further used as a counter.

File: cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js (hunk 3 above)
Comment (JafferFreshWorks) [reply cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js:8]: No

File: cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js (hunk 4 above)
Comment (JafferFreshWorks) [reply cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js]: Done.

Comment (JafferFreshWorks) [reply cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js]: Done.

**Review by junminahn — APPROVED** [review]

**Pull request merged**